internal/validate/validate.go

```diff
@@ -2,13 +2,12 @@ package validate
 
 import (
 	"fmt"
+	helpers2 "github.com/1inch/1inch-sdk-go/internal/helpers"
```

Comment:
again

Reply:
helpers will be removed at all, as well as current file